core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -153,6 +156,12 @@
     width: 100%;
   }
 
+  .conversation-skin-audio-highlight {
+    border-style: solid;
+    border-width: 3px;
+    border-color: #f3d140
```

Comment:
missing semicolon

I think you can just do `border: 3px solid #f3d140;`

Reply:
done